src/averageAges.js

```diff
@@ -14,12 +14,16 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = 0) {
+  return people
+    .filter(el => {
+      return century
+        ? el.sex === 'm' && Math.ceil(el.died / 100) === century
+        : el.sex === 'm';
+    })
+    .reduce((sum, el, i, receivedArray) => {
+      return sum + (el.died - el.born) / receivedArray.length;
+    }, 0);
```

Comment:
```suggestion
    .reduce((sum, el, i, receivedArray) =>
      sum + (el.died - el.born) / receivedArray.length
    , 0);
```

Reply:
your code in reduce callback repeats, you can move it to separate function and use it